locations/storefinders/rio_seo.py

```diff
@@ -38,7 +39,14 @@ def parse_autocomplete(self, response: Response, **kwargs: Any) -> Any:
 
     def parse(self, response, **kwargs):
         map_list = response.json()["maplist"]
-        data = json.loads("[{}]".format(Selector(text=map_list).xpath("//div/text()").get()[:-1]))
+        try:
+            data = json.loads("[{}]".format(Selector(text=map_list).xpath("//div/text()").get()[:-1]))
+        except json.decoder.JSONDecodeError:
+            logging.warning("Could not parse response - check API output")
```

Comment:
Should `self.logger` be used instead of directly using `logging`, per https://docs.scrapy.org/en/latest/topics/logging.html#logging-from-spiders ?

Reply:
Done